pkl-core/src/test/files/LanguageSnippetTests/output/errors/listingTypeCheckError1.err

```diff
@@ -0,0 +1,11 @@
+–– Pkl Error ––
+Expected value of type `String`, but got type `Int`.
+Value: 1
+
+x | res = new Listing<String> {
+                      ^^^^^^
```

Comment:
TODO: we should ideally show the source section of the member that failed this check here.

Reply:
Figured it out!